<launch>


	<!--ar码的边长，单位cm-->
	<arg name="marker_size" default="5" />
	<!--摄像头的话题名-->

	<group ns="bottom_camera">
		<!--打开individualMarkersNoKinect节点并映射为ar_track_alvar节点-->
		<node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect">
			<param name="marker_size" type="double" value="$(arg marker_size)" />
			<param name="max_new_marker_error" type="double" value="0.08" />
			<param name="max_track_error" type="double" value="0.2" />
			<param name="output_frame" type="string" value="base_link" />
			<remap from="camera_image" to="image_raw" />
			<remap from="camera_info" to="camera_info" />
			<param name="special_detect" type="bool" value="true" />
			<param name="special_id" type="int" value="1" />
		</node>
	</group>
	<node pkg="ar_pose" type="pose_adjust" name="base_pose_adjust" >
		<param name="target_roll" type="double" value="-1.57" />
		<param name="target_pitch" type="double" value="-1.57" />
		<param name="target_yaw" type="double" value="0" />
	</node>
</launch>
